.github/workflows/package_for_test.yml

```diff
@@ -46,6 +46,37 @@ jobs:
         with:
           ref: refs/pull/${{ github.event.issue.number }}/merge
 
+      - name: Check without commit after comment
```

Comment:
This job name could be clearer, e.g.

`ensure no more commits after the triggering comment`